### ISSUE TITLE
BitVector.highByte returns "0x01"

### ISSUE THREAD

**Issue by soronpo** [scodec/scodec#108]
```scala
import scodec.bits._
val x: BitVector = BitVector.highByte
```

```
x: scodec.bits.BitVector = BitVector(8 bits, 0x01)
```

**Comment by mpilquist** [comment]
Fixed in scodec-bits 1.1.6 (https://github.com/scodec/scodec-bits/pull/81). Will release that version shortly.

**Comment by mpilquist** [comment]
Thanks for reporting!
